src/shared/sam/cli/samCliBuild.ts

```diff
@@ -91,6 +97,7 @@ export class SamCliBuildInvocation {
         this.addArgumentIf(invokeArgs, !!this.dockerNetwork, '--docker-network', this.dockerNetwork!)
         this.addArgumentIf(invokeArgs, !!this.useContainer, '--use-container')
         this.addArgumentIf(invokeArgs, !!this.skipPullImage, '--skip-pull-image')
+        this.addArgumentIf(invokeArgs, !!this.manifestPath, '--manifest', this.manifestPath!)
```

Comment:
samCliBuild.test.ts has tests to ensure the other args are handled properly. Can you please add similar tests for manifest path? Copying the tests related to docker network should be quick and relevant.

A commitment to doing this in a follow up PR is okay too.

Reply:
✔️